app/views/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user, html: { class: "form-horizontal" } do |f|
```

Comment:
This is no longer the right way to setup horizontal forms. It is now `layout: :horizontal`. You can see this in the docs.